web_external/js/geojsonUtil.js

```diff
@@ -114,7 +114,7 @@ minerva.geojson.normalize = function normalize(geojson) {  // eslint-disable-lin
                     normalized = $.extend({series: []}, norm);
                     normalized.summary = {};
                 }
-                var label = '' + (entry.label || entry.time || ('Frame ' + (normalized.series.length + 1)));
+                var label = '' + (entry.label || entry.time?moment(entry.time).format('L LTS'):null || ('Frame ' + (normalized.series.length + 1)));
```

Comment:
Don't you want to use `entry.label` if it is set?  This functionally groups as `(entry.label || entry.time)?moment(entry.time).format('L LTS'):null || ('Frame ' + (normalized.series.length + 1))`, whereas I think you want `entry.label || (entry.time?moment(entry.time).format('L LTS'):null) || ('Frame ' + (normalized.series.length + 1))`